templates/endpoints/README.md

```diff
@@ -151,6 +153,60 @@ Endpoints allows you to easily add new models by adding a single configuration f
 To learn more about how to customize or add new models, 
 see the [Model Registry](models/README.md).
 
+# Serving LoRA Models
+
+`serve_lora.yaml` and `query_lora.py` are provided for you in this template.
```

Comment:
We should add something to the order of - We support serving multiple LoRA adapters with a common base model in the same request batch. In addition, we use serve multiplexing to reduce the number of swaps for LoRA adapters.